11.curiosity.ipynb

```diff
@@ -0,0 +1,723 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #59.            self.use_extrinsic = use_extrinsic
</pre><p>Need to add description in the docstring</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Curt-Park/rainbow-is-all-you-need/pull/53/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>